icons/folder-sync.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../icon.schema.json",
   "contributors": [
-    "danielbayley"
+    "danielbayley",
+    "jguddas"
```

Comment:
Not sure if you made this one or is automatically added. Is this change intentional?

Reply:
It's a manuall edit, not an automatic optimization using https://lucide-studio.vercel.app.

Reply:
True but the design is not changed right?

Reply:
Not significantly.

<img width="150" alt="image" src="https://github.com/lucide-icons/lucide/assets/25524993/33a65b3c-dc97-421b-b745-f6291bfb6896">